src/components/navBar/navBar.js

```diff
@@ -115,11 +137,34 @@ function MdNavBar($mdAria) {
           '</ul>' +
         '</nav>' +
         '<md-nav-ink-bar></md-nav-ink-bar>' +
+      '</div>'+
       '</div>',
     link: function(scope, element, attrs, ctrl) {
-      if (!ctrl.navBarAriaLabel) {
-        $mdAria.expectAsync(element, 'aria-label', angular.noop);
-      }
+        if (!ctrl.navBarAriaLabel) {
```

Comment:
Indentation is off here and below. Try to keep the styles consistent throughout. (Spacing around if statements, indentation, lining up vars, etc)